lib/fauxpaas/logged_release.rb

```diff
@@ -11,10 +11,14 @@ class LoggedRelease
 
     class << self
       def from_hash(hash)
+        time = Time.strptime(hash[:time], time_format)
```

Comment:
I'm thinking a util module for these time formatting/parsing things would be good, so we just have semantic method names in one (yard-documented) place instead of scattering references to stdlib and config keys separately.